project/Dependencies.scala

```diff
@@ -1,15 +1,15 @@
 import sbt._
 
 object Dependencies {
-  val DAML_SDK_VERSION = "100.13.20"
+  val DAML_SDK_VERSION = "100.13.39"
   val scalapbVersion = "0.9.2"
   val yamlVersion = "1.23"
   val cucumberVersion = "4.3.1"
 
   lazy val scalaTest = "org.scalatest" %% "scalatest" % "3.0.5"
   lazy val damlJavaBinding = "com.daml.ledger" % "bindings-java" % DAML_SDK_VERSION
   lazy val damlLedgerClient = "com.daml.ledger" % "bindings-rxjava" % DAML_SDK_VERSION
-  lazy val damlLFArchive = "com.digitalasset" % "daml-lf-archive" % DAML_SDK_VERSION
+  lazy val damlLFArchive = "com.digitalasset" % "daml-lf-dev-archive-java-proto" % DAML_SDK_VERSION
```

Comment:
is this the same lib but with a totally new name?

Reply:
Yes, theoretically. See the DAML release notes:

v0.13.31 on renamings
com.digitalasset.daml-lf-archive-scala to com.digitalasset.daml-lf-archive-reader
v0.13.30 on renamings
com.digitalasset.daml_lf becomes com.digitalasset.daml_lf_dev
com.digitalasset.daml-lf-proto becomes com.digitalasset.daml-lf-dev-archive-proto
*com.digitalasset.daml-lf-archive* becomes com.digitalasset:daml-lf-dev-archive-java-proto